client/src/assets/scss/_new.scss

```diff
@@ -3994,6 +3998,42 @@ div.page-activity.ncd {
       }
     }
 
+    .form.referral,
+    .form.medication-distribution {
+
+      h2 {
+        text-transform: uppercase;
+      }
+    }
+
+    .form.medication-distribution {
+
+      label {
+
+        .treatment {
+          color: $color-primary;
+          font-size: 23px;
+          font-weight: 500;
+          margin-right: 10px;
+        }
+
+        .dosage {
+          font-size: 22px;
+        }
+      }
+
+      .instructions {
+        margin-bottom: 0;
+      }
+    }
+
+    .form.referral {
+
+      .separator:last-of-type {
+        display: none;
```

Comment:
Isn't it something that we could solve in Elm more or less with the same easiness?

Reply:
@AronNovak 

Well, what happens is that we have several blocks, each ending with separator.
We don't know which ones will apear.
So, we could mofify code to insert these separators between the blocks, but it's much harder than this CSS.